app/js/layout/App.jsx

```diff
@@ -1,27 +1,59 @@
 import React, { Component, Fragment } from 'react';
 import { HashRouter, Route } from "react-router-dom";
+import { connect } from 'react-redux';
 import { Container } from 'reactstrap';
 
 import Header from './Header';
 import HomeContainer from '../containers/HomeContainer';
-import HelloContainer from '../containers/HelloContainer';
 import PriceContainer from '../containers/PriceContainer';
+import LicenseContainer from '../containers/LicenseContainer';
+
+import prices from '../features/prices';
+
+const relevantPairs = {
+  from: ['ETH', 'SNT'],
+  to: ['USD']
+};
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { ready: false }
+  }
+
+  componentDidMount() {
+    this.props.fetchPrices(relevantPairs);
+
+    EmbarkJS.onReady(async (err) => {
+      if (!err) {
+        this.setState({ ready: true })
+      }
```

Comment:
Make sure we don't make request before embarkjs is ready.
I feel we can move that into global state in another PR

Agree?

Reply:
Yeah we should. We a loader in the meantime